docs/docs/upgrade/index.md

```diff
@@ -21,16 +21,38 @@ Upgrading to a newer version of Marathon should be executed in the following ord
 1. Start the instance with the new version of Marathon.
 1. Stop the last node with the old version. Now the new version of Marathon will take over leadership and becomes active.
 1. Watch the log of this instance for a successful start. There should be no ERROR or FATAL statements in the logs.
+1. Install the new version of Marathon on all remaining nodes with the old version.
 1. Start all other instances of Marathon instances to build a quorum.
 
 
-## Upgrading from 0.7.* or 0.8.* to 0.9.*
+## Upgrading to 0.13
 
-0.8.x and 0.9.x only add new optional fields and do not change the storage format in an incompatible fashion.
+Release Notes: https://github.com/mesosphere/marathon/releases/tag/v0.13.0-RC1
+
+Tasks keys and storage format in ZooKeeper changed in a backward incompatible fashion.
+Zookeeper Compression is implemented and enabled by default. Older versions will not be able to read compressed entities.
```

Comment:
I'd add the note: `We recommend a backup of the Zookeeper state before migrating.`